modules/core/src/controllers/map-controller.ts

```diff
@@ -471,6 +477,7 @@ export class MapState extends ViewState {
 export default class MapController extends Controller {
   constructor(props) {
     props.dragMode = props.dragMode || 'pan';
+    props.position = props.position || [0, 0, 0];
```

Comment:
You are requiring `position` to be present in any props object, not just the one used on initial construction. Move this to [setProps](https://deck.gl/docs/api-reference/core/controller#setpropsprops)

Reply:
Done